src/distances/mod.rs

```diff
@@ -1,3 +1,4 @@
+pub mod alignment_helpers;
```

Comment:
Make this `pub(crate)` so that users of the library do not have access to these helpers.